python/osrd_schemas/osrd_schemas/infra.py

```diff
@@ -435,6 +435,7 @@ class Sign(TrackLocationTrait):
     value: Optional[NonBlankStr] = Field(
         description="If the sign is an announcement, precise the value(s)", default=None
     )
+    kp: NonBlankStr = Field(description="Kilometric point of the sign")
```

Comment:
```suggestion
    kp: NonBlankStr = Field(description="Kilometric point of the sign", default=None)
```

This could be useful to avoid breaking python scripts using `railjson_generator` :shrug:

Reply:
Considering the field isn't optional in editoast, and extensions are ignored by core... I'm not sure it's a good idea